resources/knative-provisioner-natss/values.yaml

```diff
@@ -10,8 +10,8 @@ knative-provisioner-natss:
   controller:
     image:
       pullPolicy: IfNotPresent
-      location: gcr.io/knative-releases/github.com/knative/eventing/contrib/natss/cmd/channel_controller@sha256:943e9ba03f1f7536224f51bfb5181039c773bd4a2209574f44a385d9748ac97f
+      location: gcr.io/knative-releases/knative.dev/eventing-contrib/natss/cmd/channel_controller@sha256:3758dc790e862bac8098b469f3d37b4747fd6a5fb4e3a1f31bad1864e638c553
```

Comment:
We need to wait until https://github.com/kyma-project/kyma/issues/6310 is implemented. Otherwise we will reintroduce this bug: https://github.com/kyma-project/kyma/issues/6233